### DISCUSSION TITLE
Multiple `@teleport` directives in component behave strangely

### DISCUSSION THREAD

**Discussion by ryangjchandler** [livewire/livewire#6852] [7. Bugs (archived)]
### Livewire version

v3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a component with two separate `@teleport` blocks.
2. See that the teleports work for the first render.
3. Call an action on the component and observe that one of the teleports disappears.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by robsontenorio** [comment]
@ryangjchandler same issue. Have you solved it ?

**Reply by ryangjchandler** [reply]
In my case, both of the teleports were to the same target element, so I wrapped them in a single teleport. But there is still an issue with multiple teleports to separate targets.
